spec/controllers/credence_games_controller_spec.rb

```diff
@@ -0,0 +1,133 @@
+require 'spec_helper'
+
+describe CredenceGamesController do
+  before(:each) do
+    # Without this, the stub on CredenceGame throws an error the second time around.
+    controller.stub(:set_timezone)
+
+    @user = mock_model(User).as_null_object
+    controller.stub(:current_user).and_return(@user)
+  end
+
+  it 'should not error if the db has not been initialized' do
+    expect { get :index }.not_to raise_error
+  end
+
+  describe 'Receiving a question' do
+    before(:each) do
+      CredenceQuestion.stub(:exists?).and_return(true)
+      @game = valid_credence_game
```

Comment:
I've updated to use let (in 0675fd1), but it's not clear to me where subject would be helpful. Could you give an example for this file?

I'm also not sure that let will improve speed in this case, because the before block is accessing the variables (and I don't think I can easily avoid that).